pages/common/linode-cli.md

```diff
@@ -0,0 +1,37 @@
+# linode-cli
+
+> Manage Linode cloud services.
+> Some subcommands such as `linode-cli linodes` have their own usage documentation.
```

Comment:
All other pages have only the subcommand in this notice:

https://github.com/tldr-pages/tldr/blob/main/pages/common/git.md?plain=1#L4
> Some subcommands such as `commit` ...

I changed `linodes` to `events` because that's one of the subcommands not mentioned in examples.
```suggestion
> Some subcommands such as `events` have their own usage documentation.
```